modules/core/test/unit/bitgo.ts

```diff
@@ -279,13 +280,30 @@ describe('BitGo Prototype Methods', function() {
   });
 
   describe('ECDH sharing secret', () => {
+    function getKey(seed: string) {
+      return bitcoin.HDNode.fromSeedBuffer(
+          crypto.createHash('sha256').update(seed).digest()
+      ).keyPair;
+    }
+
     it('should calculate a new ECDH sharing secret correctly', () => {
-      const bitgo = new TestBitGo();
-      const eckey1 = bitcoin.ECPair.makeRandom({ network: bitcoin.networks[common.getNetwork()] });
-      const eckey2 = bitcoin.ECPair.makeRandom({ network: bitcoin.networks[common.getNetwork()] });
-      const sharingKey1 = bitgo.getECDHSecret({ eckey: eckey1, otherPubKeyHex: eckey2.getPublicKeyBuffer().toString('hex') });
-      const sharingKey2 = bitgo.getECDHSecret({ eckey: eckey2, otherPubKeyHex: eckey1.getPublicKeyBuffer().toString('hex') });
-      sharingKey1.should.equal(sharingKey2);
+      for (let i=0; i<256; i++) {
+        const bitgo = new TestBitGo();
```

Comment:
do we need to instantiate 256 `TestBitGo` objects?

Reply:
we don't - actually `bitgo.getECDHSecret` should at least be a static class method, probably just a regular method instead